src/scss/components/_news-and-views.scss

```diff
@@ -131,6 +131,37 @@
 	fill: var(--theme-background-fill, #6895c6);
 }
 
+// Initially show the static generated page and hide the dynamic page section for the search and filtering result.
+.views.dynamic-view {
+	display: none;
+}
+
+.views {
+	.search-container {
+		padding-bottom: rem(27);
+
+		svg {
+			margin-left: rem(16);
+		}
+
+		form {
+			flex: 1;
+		}
+
+		input[type="search"] {
+			border: 1px solid $green-dark;
```

Comment:
This line is necessary for the outline of the view search bar to appear in the default theme but causes the outline to be "extra" thick in uio contrast themes as shown in the image below.

<img width="537" alt="Screen Shot 2020-06-23 at 1 06 33 PM" src="https://user-images.githubusercontent.com/25180846/85464053-10b2e680-b55c-11ea-9691-c36640edb0fd.png">

This is because both `border: 1px solid $green-dark;` and `box-shadow: 0 0 0 rem(2) currentColor inset;` are taking place in uio contrast themes.

I would suggest adding the following code or something similar to the _mixins.scss file:
```{css}
.views .search-container input[type=search] {
   border: none;
}
```

This way only `box-shadow: 0 0 0 rem(2) currentColor inset;` will take place when uio contrast themes are turned on.

Reply:
Really nice catch, @TeddTech. It has been addressed and ready for another look.